app/src/main/java/com/example/secondtask_composecalculator/CalculatorButtonComponent.kt

```diff
@@ -0,0 +1,76 @@
+package com.example.secondtask_composecalculator
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import com.example.secondtask_composecalculator.data.ActionEnum
+import com.example.secondtask_composecalculator.ui.theme.*
+
+
+@Composable
+fun getButtonColor(buttonSymbol: ActionEnum): Color{
```

Comment:
Что-то я не заметил, где используются эта функция и следующая, они точно нужны? И тем более точно ли они должны быть Composable?

Reply:
@Chantreck Первая меняет по символу из кнопки цвет самой кнопки, а вторая изменяет цвет кнопки удаления, когда поле для чисел полностью пустое
```suggestion
ButtonModel(
    buttonSymbol = buttonSymbol,
    onClick = { handleButtonClick(buttonSymbol, expression) },
    color = getButtonColor(buttonSymbol),
    fontSize = fontSize
    )
```